msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -672,10 +842,12 @@ private void acquireToken(@NonNull final Activity activity,
                 .withClaims(claimsRequest)
                 .build();
 
-        acquireTokenAsync(acquireTokenParameters);
+
+
+        acquireToken(acquireTokenParameters);
     }
 
-    private static void validateNonNullArgument(@Nullable Object o,
+      protected static void validateNonNullArgument(@Nullable Object o,
```

Comment:
May this method can be used for all the `context` null checks above introduced

Reply:
Makes sense